src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -297,7 +302,12 @@ function SubmissionsFactory(
               const { data } = event
               const { csvRecord, downloadBlob } = data
 
-              if (csvRecord.status === 'ERROR') {
+              if (csvRecord.status === 'ATTACHMENT_ERROR') {
+                attachmentErrorCount++
+                errorCount++
+                // accumulate dataset anyway, since the download is otherwise OK
```

Comment:
we also want to accumulate the submission ID and timestamp, if the decryption fails

Reply:
Changed